### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Goruchie**
# **Add the following features:**

- **Add any missing JSX elements to the project (without adding extra functionality)**
- **Use the CSS rules detailed in Zeplin to style the website.**
- **The final design should match Zeplin's design (this includes all elements + the same styling)**

### CODE REVIEW

File: src/modules-css/book.css [hunk 1]
```diff
@@ -103,6 +105,7 @@
 
 .main-ul {
   list-style-type: none;
+  margin-left: 25px;
```
Comment (emmiiorji) [inline src/modules-css/book.css:113]: Your design looks great. For it to match the design, there are some adjustments needed to be made. Note the following points:

- The shade of the blue colors to match the used color for the page title, progress bars, add book button and update button.
- The remove button has no background and borders
- A uniform left and right margins are expected between the book cards and their container
- The title of the page is left aligned with the book cards
- There's no horizontal rule between the cards and the margin between the cards are bigger in your design than they appear on the design guide.
- Between the Books and Categories, the current page has a different darker color
- The category for each book appears above the name

Your design:
image

Design guideline:
image

Kindly adjust the accordingly.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Goruchie, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

a little more

## Good Highlights✔

- Linter checks are passing
- Adhered to Gitflow
- Repo is professional


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules-css/book.css (hunk 1 above)
Comment (Goruchie) [reply src/modules-css/book.css:113]: Thank you for the review! I resolve all the issues, so now I must be approved :)
PD: I have some electriciry issues, so please approve me :(
image

**Comment by emmiiorji** [comment]
Hi @Goruchie 

Great improvement there, kudos.
Kindly request for another review on your dashboard and the next reviewer will do justice.

**Comment by emmiiorji** [comment]
Hi @Goruchie 

Great improvement there, kudos.
Kindly request for another review on your dashboard and the next reviewer will do justice.

**Comment by emmiiorji** [comment]
Hi @Goruchie 

Great improvement there, kudos.
Kindly request for another review on your dashboard and the next reviewer will do justice.

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @Goruchie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**